pymove/utils/geoutils.py

```diff
@@ -0,0 +1,308 @@
+import geohash2 as gh
+import numpy as np
+
+from pymove.utils.constants import (
+    BASE_32,
+    BIN_GEOHASH,
+    COLOR,
+    GEOHASH,
+    LATITUDE,
+    LATITUDE_DECODE,
+    LONGITUDE,
+    LONGITUDE_DECODE,
+)
+from pymove.utils.log import progress_bar
+
+
+def v_color(ob):
+    """
```

Comment:
Codacy found an issue: [1 blank line required between summary line and description (found 0) (D205)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5971443)